modules/client/src/lib/index.ts

```diff
@@ -1,4 +1,3 @@
-export * from "./bn";
```

Comment:
Is this necessary to be removed? It will break Metamask integration

Reply:
Added back